foreman/foreman.spec

```diff
@@ -570,6 +588,8 @@ rm -rf %{buildroot}
 %config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
 %config %{_sysconfdir}/cron.d/%{name}
 %{_sysconfdir}/rpm/macros.%{name}
+%{_sysconfdir}/rpm/macros.%{name}-apipie
+%{_sysconfdir}/rpm/macros.%{name}-assets
```

Comment:
Ahh I see now. I'd prefer to merge assets and apipie macros to macros.foreman-plugin and ship that in plugin subpackage with the sqlite dependency. I'll update the PR that way if there are no objections @domcleal.